index.bs

```diff
@@ -1931,8 +1931,11 @@ To <dfn for="WebTransportError">set up</dfn> a {{WebTransportError}} |error| wit
 *This section is non-normative.*
 
 This section describes the [[QUIC]] protocol behavior of methods defined
-in this specification, utilizing [[WEB-TRANSPORT-HTTP3]]. Cause and effect may
-not be immediate due to buffering.
+in this specification, utilizing [[WEB-TRANSPORT-HTTP3]]. The application
+{{WebTransportError/streamErrorCode}} in the {{WebTransportError}} error is
+converted to an httpErrorCode, and vice versa, as specified in [[!WEB-TRANSPORT-HTTP3]]
+[section 4.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.3).
+Cause and effect may not be immediate due to buffering.
```

Comment:
I'm wondering, since the added text is specific only to a subset of methods, if we should add it to the end instead, so the "cause and effect" statement cannot be misread as applying only to the subset?

Doesn't read as well though. WDYT?
```suggestion
in this specification, utilizing [[WEB-TRANSPORT-HTTP3]]. Cause and effect may
not be immediate due to buffering. The application
{{WebTransportError/streamErrorCode}} in the {{WebTransportError}} error is
converted to an httpErrorCode, and vice versa, as specified in [[!WEB-TRANSPORT-HTTP3]]
[section 4.3](https://datatracker.ietf.org/doc/html/draft-ietf-webtrans-http3/#section-4.3).
```

Reply:
Hmmm, I agree that it doesn't read as well, but I can also see why one might misread it that way. I don't have strong opinions, so I'll add the text at the end instead as you suggested.